tools/ansible/roles/dockerfile/templates/supervisor_task.conf.j2

```diff
@@ -31,7 +31,6 @@ command = awx-manage run_wsrelay
 directory = /var/lib/awx
 {% endif %}
 autorestart = true
-startsecs = 30
```

Comment:
What was this used for?

Reply:
From [docs](http://supervisord.org/configuration.html): 
> The total number of seconds which the program needs to stay running after a startup to consider the start successful (moving the process from the STARTING state to the RUNNING state). Set to 0 to indicate that the program needn’t stay running for any particular amount of time.

> Note

> Even if a process exits with an “expected” exit code (see exitcodes), the start will still be considered a failure if the process exits quicker than startsecs.

> Default: 1